src/status_im/chat/models/reactions.cljs

```diff
@@ -105,3 +107,42 @@
        acc))
    []
    reactions))
+
+(defn- <-rpc
+  [emoji-reaction]
+  (-> emoji-reaction
+      (set/rename-keys
+       {:emojiId       :emoji-id
+        :compressedKey :compressed-key})
+      (select-keys [:emoji-id :from :compressed-key])))
+
+(defn- format-response
+  [response]
+  (->> (transforms/js->clj response)
+       (map <-rpc)
+       (group-by :emoji-id)
+       (into (sorted-map))))
+
+(rf/defn save-emoji-reaction-details
+  {:events [:chat/save-emoji-reaction-details]}
+  [{:keys [db]} message-reactions long-pressed-emoji]
+  {:db (-> db
+           (assoc-in [:chats (:current-chat-id db) :message/reaction-authors-list] message-reactions)
+           (assoc-in [:chats (:current-chat-id db) :message/reaction-authors-list-selected-reaction]
```

Comment:
a few questions
1) why we store this in chat map
2) `:message/` usage inside chat map looks strange
3) can the name be shortened?

so i would say we should have a map, reaction-authors, when we will store, list of authors and selected reaction

Reply:
Made it a root level subscription and shortened the name by using a map like you suggested, and renamed.